compose/cli/main.py

```diff
@@ -268,6 +268,7 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
+            --skip-missing          Skip unpullable images.
```

Comment:
Hmm, it seems like you aren't really skipping as much as ignoring.

`--ignore-pull-failures  Builds what it can and ignores images with pull failures.`

I also like `--continue-on-error` if it really continues everything regardless of error...but if they operation can error out it isn't really true.